set.mm

```diff
@@ -804812,6 +804834,214 @@ edge function instead of the edges themselves (see also ~ clnbgrval ).
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Semiclosed and semiopen neighborhoods (experimental)
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  We have already definitions for open and closed neighborhoods of a vertex,
+  which differs only in the fact that the first never contains the vertex,
+  and the latter always contains the vertex. One of these definitions, however,
+  cannot be simply derived from the other.  This would be possible if a
+  definition of a semiclosed neighborhood was available, see ~ dfsclnbgr2 .
+  The definitions for open and closed neighborhoods could be derived from such
+  a more simple, but otherwise probably useless definition, see ~ dfnbgr5 and
+  ~ dfclnbgr5 . Depending on the existence of certain edges, a vertex belongs
+  to its semiclosed neighborhood or not.
+
+  An alternate approach coukd have been to introduce semiopen neighborhoods,
+  see ~ dfvopnbgr2 .  The definitions for open and closed neighborhoods could
+  also be derived from such a definition, see ~ dfnbgr6 and ~ dfclnbgr6 .  As
+  for the definition of a semiclosed neighborhood, it is unclear if this
+  definition is/will be useful, and in contrast to ~ dfsclnbgr2 , it is much
+  more complex.  Also depending on the existence of certain edges, a vertex
+  belongs to its semiopen neighborhood or not.
```

Comment:
```suggestion
  also be derived from such a definition, see ~ dfnbgr6 and ~ dfclnbgr6 .
  Like with semiclosed neighborhoods, depending on the existence of certain
  edges, a vertex belongs to its semiopen neighborhood or not.

  It is unclear if either definition is/will be useful, and in contrast to
  ~ dfsclnbgr2 , the definition of semiopen neighborhood is much more
  complex.  
```
suggestion, basically this extracts the second last sentence to make it more clear, changing specifically the `As for the definition of a semiclosed neighborhood` part